### PR TITLE
Plain Button Unit Tests

### LINKED ISSUE #37: Tests Written for PlainButton
As a developer, it would be helpful to know that we have not broken any existing features. In order to do this, we want to have as much reasonable test coverage as possible. 

Acceptance Criteria: 
- Unit tests written for PlainButton Component
- Tests pass

### PR DESCRIPTION
**Pull request by natoh19**
# Summary

Resolves #37 
Two basic unit tests

## What was changed?

Created PlainButton folder for the two-unit tests added

### CODE REVIEW

**Review by jufoobar — COMMENTED** [review]
I would remove the things that I mentioned, but other than that it looks good and the tests work :)

**Comment by OleDakotaJoe** [comment]
General comment: 
when we are writing tests, can we keep the tests to a "given___when___then___" naming convention? 
THis isn't some sort of hard requirement, and I'd gladly accept a "no" with an explanation. 

I personally just like naming tests this way, because I can generally read the test report, and know what I did that broke the test before I ever even click through to the test.

**Comment by jufoobar** [comment]
> General comment: when we are writing tests, can we keep the tests to a "given___when___then___" naming convention? THis isn't some sort of hard requirement, and I'd gladly accept a "no" with an explanation.
> 
> I personally just like naming tests this way, because I can generally read the test report, and know what I did that broke the test before I ever even click through to the test.

I like this as well 👍

File: src/components/Buttons/PlainButton/PlainButton.test.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import * as React from 'react';
+import '@testing-library/jest-dom';
+import userEvent from '@testing-library/user-event';
+import { fireEvent, render, screen, waitFor } from '@testing-library/react';
+
+import { PlainButton } from './PlainButton.component';
+import { Colors } from '../../../config';
```
[the remark this answers is not in the document]
Comment (natoh19) [reply src/components/Buttons/PlainButton/PlainButton.test.tsx]: done

File: src/components/Buttons/PlainButton/PlainButton.test.tsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import * as React from 'react';
+import '@testing-library/jest-dom';
+import userEvent from '@testing-library/user-event';
+import { fireEvent, render, screen, waitFor } from '@testing-library/react';
+
```
[the remark this answers is not in the document]
Comment (natoh19) [reply src/components/Buttons/PlainButton/PlainButton.test.tsx]: removed

File: src/components/Buttons/PlainButton/PlainButton.test.tsx [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    test('its position should be set to relative and it should have expected transition by default', () => {
+        const expectedCss = 'position: relative;transition: all 0.2s;';
+        render(<PlainButton children={'PlainButton'} />);
+        expect(screen.getByRole('button', { name: 'PlainButton' })).toHaveStyle(
+            expectedCss
+        );
+    });
+
+    test('if disabled attribute passed in should be disabled', () => {
+        const expectedCss = 'position: relative;transition: all 0.2s;';
```
[the remark this answers is not in the document]
Comment (natoh19) [reply src/components/Buttons/PlainButton/PlainButton.test.tsx]: removed

**Comment by natoh19** [comment]
> General comment: when we are writing tests, can we keep the tests to a "given___when___then___" naming convention? THis isn't some sort of hard requirement, and I'd gladly accept a "no" with an explanation.
> 
> I personally just like naming tests this way, because I can generally read the test report, and know what I did that broke the test before I ever even click through to the test.

changed

**Review by OleDakotaJoe — APPROVED** [review]
Love it. Thanks for the great work

**Comment by OleDakotaJoe** [comment]
This looks great. Once you get the merge conflicts resolved, will give it a merge

**Pull request merged**